malign_buffer.cc

```diff
@@ -114,6 +118,13 @@ size_t MalignBuffer::RoundUpToPageSize(size_t k) {
   return ((k + kPageSize - 1) / kPageSize) * kPageSize;
 }
 
+size_t CacheLineSize() {
+    size_t lineSize = 0;
```

Comment:
I think this needs to be in the apple #ifdef , might as well put it where it's used in that case

Reply:
i updated see below